index.html

```diff
@@ -6,14 +6,151 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <!-- <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+        integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous"> -->
+    <link rel="stylesheet" type="text/css" href="css/main.css">
     <!-- Add a link to your custom CSS here -->
 
-    <title>Responsive grid project</title>
+    <title>Portfolio - Responsive grid project</title>
 </head>
 
 <body>
-
     <!-- Add your markup here -->
+    <main>
+        <section class="mainpage-section">
+            <div class="container">
+                <div class="row">
+                    <div class="col-12 sm-col-12 md-col-12 lg-col-12 xl-col-12">
+
+                        <div class="logo-wrapper">
+                            <img src="img/logo_b_t.png" class="logo" alt="B logo" title="B logo">
+                        </div>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-12 sm-col-12 md-col-12 lg-col-12 xl-col-12 center">
+                        <nav>
```

Comment:
if the number of columns is the same across breakpoints, you don't have to repeat them for each size.
`col-12` works from tiny screen up to infinity - this is the 'mobile-first' approach. So you could just have `col-12` here and it would work the same 😄 
You only need to use the other sizes if you're changing the number of columns on different breakpoints (example mobile 12, then tablet 6 then desktop 4 columns, you would need different classes for each of those, like this: `col-12 md-col-6 lg-col-4`).

Reply:
Also here you could make that `div` on line 31 the `nav` instead, and that way not have unnecessary wrappers.